app/models/history.rb

```diff
@@ -449,8 +451,9 @@ def self.follow_up_flag_edit(history)
   def self.clear_follow_up_flag(history)
     return if history[:follow_up_reason_before].nil?
 
-    comment = 'User cleared flag for follow-up.'
-    comment += " Reason: #{history[:clear_flag_reason]}" unless history[:clear_flag_reason].blank?
+    comment = 'User cleared flag for follow-up'
+    comment += compose_explanation(history)
+    comment += ". Reason: #{history[:clear_flag_reason]}" unless history[:clear_flag_reason].blank?
```

Comment:
The way this is organized, if the monitoree has their flag cleared without a reason given, the resulting message is missing a `.`:
![image](https://user-images.githubusercontent.com/52747882/125354224-8fc6b880-e331-11eb-8966-63c48f9ac59a.png)